### ISSUE TITLE
Specify target framework when discovering tests

### ISSUE THREAD

**Issue by seesharper** [formulahendry/vscode-dotnet-test-explorer#136]
First of all, thanks for this awesome extension. Great work 👍 

Would it be possible to somehow specify the target framework when doing test discovery?

Either group them in the explorer by target framework or simply make it a configuration value that can be passed to `dotnet test -f netcoreapp2.0 -t -v=q`. Right Now you can't really `Go to test` since there are multiple matching symbols when you have a test project that is cross compiled.

**Comment by stefanforsberg** [comment]
Thanks! <3

We have a setting called testArguments that are added to the discovery and test running, can you try that and see if that fits your needs?

**Comment by seesharper** [comment]
Hi and thanks for such a quick response. 

Added this setting 

```
"dotnet-test-explorer.testArguments": "-c release -f netcoreapp2.0"
```

That solved the problem with multiple tests with the same name coming up in the explorer.
It did not however solve the issue with `Go to test`.  

![image](https://user-images.githubusercontent.com/1034073/45044939-d1a75e80-b071-11e8-9bbc-970b1db0450f.png)

Anything else I can try to make work? 

Again thanks a lot for your help.

**Comment by stefanforsberg** [comment]
How that works internally is that we ask the symbol server for all symbols matching the method that you click. Let's say your test you click is called is called ShouldRespondWith200OK.  If you have multiple (test) methods named that the symbol server will return a list with the method name as well as an URI to the file the method was found in.

What we try to do then is to navigate to the one which uri matches the FQN of the method you clicked. That is, for it to navigate to the correct method your folder structure needs to match the namespace exactly, otherwise we give up and show that message.

It's crude for sure, but that's how it works and I'm not sure how much better it could be given what the symbol server returns to us.

**Comment by stefanforsberg** [comment]
In your case it would probably work okay since they are the same method. I fear that for other users that maybe have the same test names in different parts of the code might be annoyed and/or confused that it navigates to the wrong one... so the current behavior is far from ideal as well.

Maybe we should do it like vscode handles it itself, like if you do a ctrl-t and search for the test method it will show a list of the matching symbols and you get to choose which is the one you want to navigate to it. How do you feel about that solution?

**Comment by seesharper** [comment]
We could check the location of the duplicate symbols. If they are the same, they all point to the same methods. Otherwise continue to show the error message or even better let the user choose 👍

**Comment by seesharper** [comment]
@stefanforsberg What do you say? Would you be accepting a PR that takes the location into the equation when resolving unique symbols?

**Comment by stefanforsberg** [comment]
@seesharper Most def!

**Comment by stefanforsberg** [comment]
Should be fixed with 0.5.4
